includes/wc-product-functions.php

```diff
@@ -375,6 +375,7 @@ function wc_scheduled_sales() {
 	// Sales which are due to start
 	$product_ids = $data_store->get_starting_sales();
 	if ( $product_ids ) {
+		do_action('wc_before_products_starting_sales', $product_ids);
```

Comment:
Looks good. Just needs formatting changes. There should be a space between the parentheses and the parameters (https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage)